src/i18n/pt-br/nav.ts

```diff
@@ -29,6 +29,7 @@ export default NavDictionary({
 	examples: 'Receitas',
 	'guides/migrate-to-astro': 'Migre para o Astro',
 	'guides/cms': 'Conecte um CMS',
+	'guides/backend': 'Adicione serviços back-end',
```

Comment:
Gramaticalmente devemos adicionar "de" aqui, como foi feito anteriormente na página.
```suggestion
	'guides/backend': 'Adicione serviços de backend',
```